charts/libvirt/templates/daemonset-libvirt.yaml

```diff
@@ -299,8 +299,6 @@ spec:
               {{- if or ( gt .Capabilities.KubeVersion.Major "1" ) ( ge .Capabilities.KubeVersion.Minor "10" ) }}
               mountPropagation: Bidirectional
               {{- end }}
-          args:
-            - "--libvirt.nova"
```

Comment:
I think we need a patch to keep this.

Reply:
moved to a local patch

Reply:
https://review.opendev.org/c/openstack/openstack-helm-infra/+/925930